lib/controllers/situations.js

```diff
@@ -68,3 +68,74 @@ exports.openfiscaResponse = function(req, res, next) {
         res.send(result);
     });
 };
+
+function createTestContent(situation, test) {
+    var openfiscaRequest = mapping.buildOpenFiscaRequest(situation, true);
+    var testCase = openfiscaRequest.scenarios[0].test_case;
+    return _.assign(test, {
+        period: openfiscaRequest.scenarios[0].period,
+        absolute_error_margin: 10,
+        individus: testCase.individus,
+        familles: testCase.familles[0],
+        foyers_fiscaux: testCase.foyers_fiscaux[0],
+        menages: testCase.menages[0]
+    });
+}
+
+function myToString(obj) {
+    return obj.toString();
+}
+
+function normalizeTestContent(testFileContent) {
+    testFileContent.individus = testFileContent.individus.map(function(individu) {
+        individu.id = individu.id.toString();
+        return individu;
+    });
+
+    function replaceElementsWithStringRepresentation(obj) {
+        function convert(leafName) {
+            obj[leafName] = obj[leafName].map(myToString);
+        }
+        return convert;
+    }
+
+    var sources = {
+        familles: ['parents', 'enfants'],
+        foyers_fiscaux: ['declarants', 'personnes_a_charge'],
+        menages: ['enfants'],
```

Comment:
This is fully constant, and should at least be capitalised. Moreover, declaring it inside the parsing function has two costs:

1. Readability: a longer function body unrelated to its algorithmic complexity.
2. Performance: you are instantiating a new version of this constant on each run.

I would recommend defining it on the top of the file, and improving its name (“sources” of what? `EXPORTED_PROPERTIES`? `PROPERTIES_TO_NORMALIZE`?).